elastic/beats
# Make cockroachdb GA

- Enhancement


## What does this PR do?

This PR make cockroachdb module GA. 

## Why is it important?



## Checklist



## How to test this PR locally
Run CockroachDB instance.
Enable cockroachdb module and update cockroachdb.yml with host and port details.
Start metricbeat and metrics should start flowing through ES.



## Related issues




- Relates #25622
- Relates #19517
- Relates https://github.com/elastic/integrations/issues/3550
